src/main/java/net/openhft/chronicle/queue/impl/single/SingleChronicleQueue.java

```diff
@@ -463,20 +460,6 @@ public ExcerptAppender acquireAppender() {
         return res;
     }
 
-    /**
-     * @return the {@link QueueLock} This lock is held while the queue replication cluster is back-filling.
-     * By Back-filling we mean that, as part of the fail-over process a sink, may actually have more data than a source,
-     * hence we need to back copy data from the sinks to the source upon startup.
-     * While we are doing this we lock the queue so that new appenders can not be created.
-     * <p>
-     * Queue locks have no impact if you are not using queue replication because the are implemented as a no-op.
-     */
-    @Override
-    @NotNull
-    public QueueLock queueLock() {
```

Comment:
Note that this was deprecated - I just un-deprecated the other day as we feared it was too soon